src/System.Private.ServiceModel/tests/Common/Infrastructure/src/CertificateManager.cs

```diff
@@ -73,7 +73,7 @@ private static StoreLocation PlatformSpecificRootStoreLocation
                 using (X509Store store = new X509Store(storeName, storeLocation))
                 {
                     store.Open(OpenFlags.ReadOnly);
-                    resultCert = CertificateFromThumbprint(store, certificate.Thumbprint);
+                    resultCert = CertificateFromThumbprint(store, certificate.Thumbprint, validOnly: false);
```

Comment:
It is worth observing that Unix considers the root cert invalid when retrieving it by thumbprint, and we still don't understand why.  Issue #1347 tracks this.  But with the changes in this PR, this still allows all the cert tests to attempt to run if an "invalid" root cert is in the store.  Only one infrastructure test is disabled currently -- on Unix, the one that tries to verify that the root cert is valid is disabled by #1347.